recipes-bsp/u-boot/files/0001-Environment-Openvario-mainline.patch

```diff
@@ -80,8 +80,8 @@ index 9819d99..76e7a83 100644
 +			"if test $rotation -eq '1'; then setenv rotation_kernel 3;fi;" \
 +			"if test $rotation -eq '3'; then setenv rotation_kernel 1;fi;" \
 +			"if test $rotation -eq '0' -o $rotation -eq '2'; then setenv rotation_kernel $rotation;fi;" \
-+			"setenv bootargs console=ttyS0,115200 root=/dev/mmcblk0p2 rootwait panic=10 fbcon=rotate:${rotation_kernel};" \
-+		"else setenv bootargs console=ttyS0,115200 root=/dev/mmcblk0p2 rootwait panic=10;" \
++			"setenv bootargs console=ttyS0,115200 root=/dev/mmcblk0p2 rootwait panic=10 lima.job_hang_limit=10 fbcon=rotate:${rotation_kernel};" \
++		"else setenv bootargs console=ttyS0,115200 root=/dev/mmcblk0p2 rootwait panic=10 lima.job_hang_limit=10;" \
```

Comment:
I'd really wish we would preserve the reason for this `lima.job_hang_limit=10` somewhere. Not sure what's the best way to do this, but it is getting harder and harder to trace it back to #71.

Reply:
Agree, that's important, I'll move the code comment over into this u-boot patch.
Unfortunately, u-boot scripts are already unwieldy; then you have a u-boot script inside a C string literal, and if this weren't complicated enough, this u-boot script inside a C string literal is then inside a patch file. This needs to improve. There should be a plain text file with the script.